packages/start-frontend/src/helpers/prompt-clack.ts

```diff
@@ -0,0 +1,193 @@
+// to execute this file
+// run node ./packages/start-frontend/src/helpers/prompt-clack.js
+
+import {
+  intro,
+  outro,
+  spinner,
+  text,
+  select,
+  group,
+  multiselect,
+  confirm,
+  cancel,
+} from "@clack/prompts";
+import * as color from "picocolors";
+import { setTimeout as sleep } from "node:timers/promises";
+import gradient = require("gradient-string");
+
+export type UserInputTests = {
+  useVitest: boolean | undefined;
+  useStorybook: boolean | undefined;
+  useE2E: boolean | undefined;
+  useEslint: boolean | undefined;
+  usePrettier: boolean | undefined;
+};
+
+interface CLIOptions {
+  [key: string]: {
+    name: string;
+    apiSolution: { name: string; value: "graphql" | "restful" }[];
+    useModules: { name: string; value: string; label: string }[];
+  };
+}
+
+const CLIOptions: CLIOptions = {
+  react: {
+    name: "React",
+    apiSolution: [
+      { name: "RESTful", value: "restful" },
+      { name: "GraphQL", value: "graphql" },
+    ],
+    useModules: [{ name: "Add CRUD Operations", value: "crud", label: "CRUD" }],
+  },
+};
+
+// ML Colors
+const g = gradient("#53575a", "#53575a");
+const t = gradient("#53575a", "#ffff00");
+
+export async function promptClack(dir) {
+  // intro example
+  intro(
+    `${g("ꮙ START-")}${t(
+      "FRONTEND"
+    )} \n\n Welcome to the frontend setup wizard!`
```

Comment:
Is this officially (?) called the frontend setup wizard now? 😄 
(It sounds very official and cool)